MANIFEST.in

```diff
@@ -0,0 +1,15 @@
+include README.rst
+include LICENSE.txt
+include CHANGELOG.md
+
+recursive-include tests *
+
+recursive-exclude * __pycache__
+recursive-exclude * *.py[co]
+
+recursive-include doc Makefile make.bat *.conf *.css *.html *.ico *.ipynb *.js *.json *.npy *.png *.py *.rst *.svg
+prune doc/builder
+prune doc/nbpublisher
+recursive-exclude doc/test_data *
```

Comment:
So I couldn't figure out how to keep this directory empty. Seems it needs at least one file to stick around. Seems to only have a README in it in `master`. Maybe we can exclude all data or just include the README?

Currently the code here deletes that directory entirely.

Reply:
I think that is fine.

Reply:
Sorry I couldn't help myself. Kept playing with it locally. Now it just keeps the README. Can revert back to this change if you prefer though.

Reply:
Also good 👍

Reply:
I don't think that's an issue. The user can't run the tests that would use this directory without the submodules anyway and we'll soon be replacing this machinery entirely.

Reply:
Alright feel free to override whatever sits on this line. I have no strong opinions. 😄